clang/test/SemaSYCL/reqd-sub-group-size-host.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsycl-is-host -fsyntax-only -verify %s
+// RUN: %clang_cc1 -fsycl-is-host -Wno-error=deprecated-declarations -fsyntax-only -verify %s
 // expected-no-diagnostics
 
 [[intel::reqd_sub_group_size(8)]] void fun() {}
```

Comment:
Once again, this is some sort of legacy. The main point of the test is to make sure that the attribute is not diagnosed as unknown during host compilation. Let's update the test to use `sycl::` attribute

Reply:
Done